R/toml_serialization.R

```diff
@@ -0,0 +1,219 @@
+#' Convert R `list()` into toml-compatible format.
+#'
+#' [to_toml()] can be used to build `Cargo.toml`.
+#'
+#' @param ... A list from which toml is constructed.
+#'     Supports nesting and tidy evaluation.
+#' @param .str_as_literal Logical indicating wether to treat
+#'     strings as literal (single quotes no escapes) or
+#'     basic (ecaping some sequences) ones. Default is `TRUE`.
+#' @param .format_int,.format_dbl Character scalar describing
+#'     number formatting. Compatible with `sprintf`.
+#' @return A character vector, each element corresponds to
+#'     one line of the resulting output.
+#' @examples
+#' to_toml(workspace = )
+#' # Produces [workspace] with no children
+#'
+#' to_toml(patch.crates_io = list(`extendr-api` = list(git = "git-ref")))
+#' # [patch.crates_io]
+#' # extendr-api = { git = 'git-ref' }
```

Comment:
Remove the output. It'll be generated automatically when the website is built. See e.g. here: https://dplyr.tidyverse.org/reference/mutate.html#examples